src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,6 +56,13 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
+        try {
+            if (dateParams.length == 3) {
+                return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+            }
+        } catch (DateTimeException e) {
```

Comment:
let's add ArrayIndexOutOfBoundException to catch and remove if upon

Reply:
done